app/src/pages/settings/DeleteUserDialog.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useCallback } from "react";
+import { graphql, useMutation } from "react-relay";
+
+import { Button, Dialog, Flex, Text, View } from "@arizeai/components";
+
+import { useNotifyError, useNotifySuccess } from "@phoenix/contexts";
+
+export function DeleteUserDialog({
+  userId,
+  onDeleted,
+  onClose,
+}: {
+  userId: string;
+  onDeleted: () => void;
+  onClose: () => void;
+}) {
+  const [commit, isCommitting] = useMutation(graphql`
+    mutation DeleteUserDialogMutation($input: DeleteUsersInput!) {
+      deleteUsers(input: $input)
+    }
+  `);
+
+  const notifySuccess = useNotifySuccess();
+  const notifyError = useNotifyError();
+
+  const handleDelete = useCallback(() => {
+    commit({
+      variables: {
+        input: {
+          userIds: [userId],
+        },
+      },
+      onCompleted: () => {
+        notifySuccess({
+          title: "User deleted",
+          message: "User has been deleted.",
+        });
+        onDeleted();
+        onClose();
+      },
+      onError: (error) => {
+        notifyError({
+          title: "Failed to delete user",
+          message: error.message,
+        });
+      },
+    });
+  }, [commit, notifyError, notifySuccess, onClose, onDeleted, userId]);
+  return (
+    <Dialog title="Delete User" isDismissable onDismiss={onClose}>
+      <View padding="size-200">
+        <Text color="danger">
+          {`Are you sure you want to delete this user? This action cannot be undone. The user will be permanently blocked, and cannot be reactivated with the same email or username.`}
```

Comment:
I think we can just remove the "blocked" wording for now

Reply:
sounds good